chart/compass/charts/provisioner/values.yaml

```diff
@@ -3,8 +3,6 @@ deployment:
   image:
     pullPolicy: IfNotPresent
   
-  resources: {}
```

Comment:
I think we are using it in the Deployment https://github.com/kyma-incubator/compass/pull/1397/files#diff-4b7a93aa1a609a4734f843e3a6f98875L41-L42 (line 42)

Reply:
Sure! My fault. It was broken anyway - it should be `.Values.deployment.resources`. I fixed it already. Same situation in the Connector chart